scripts/validate/yaml-lint

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+
```

Comment:
```
set -euo pipefail
```
Is echo'd directly to the terminal here, this doesn't change the execution model of the bash script so I've removed.